### ISSUE TITLE
Add Support for Chatbot Policies in the organizations-config

### ISSUE THREAD

**Issue by turqual** [awslabs/landing-zone-accelerator-on-aws#614]
**Is your feature request related to a problem? Please describe.**
Similar to the support for Tag and Backup policies, we would like to be able to manage our ChatBot policies from the `organizations-config.yaml` withing LZA. 

**Describe the feature you'd like**
So like we have...
```
taggingPolicies:
  - name: TagPolicy
    description: Organization Tagging Policy
    policy: tagging-policies/org-tag-policy.json
    deploymentTargets:
        organizationalUnits:
          - Root
  ```        
  ...we would like...
  ```
  chatbotPolicies:
  - name: ChatbotPolicy
    description: Organization Chatbot Policy
    policy: chatbot-policies/org-chatbot-policy.json
    deploymentTargets:
        organizationalUnits:
          - Root
  ```

This is similar to https://github.com/awslabs/landing-zone-accelerator-on-aws/issues/107 which requests the same for the `ai-opt-out` policies within Organizations.

**Comment by bo1984** [comment]
Hi Chris!

I went ahead and opened a feature request for this issue to be natively supported in LZA. As of right now, we can handle this via the customizations layer. I will leave this ticket open to keep you up to date on when this request is natively supported and released publicly. Please let us know if you have any questions or concerns in the meantime.

**Comment by erwaxler** [comment]
Hi @turqual , this feature has been included in the v1.11.0 release of the LZA published today! I will go ahead and close this issue, thanks again for your support of the Landing Zone Accelerator!
